sound/hda/trace.h

```diff
@@ -65,7 +65,7 @@ DECLARE_EVENT_CLASS(hdac_stream,
 		__entry->stream_tag = (azx_dev)->stream_tag;
 	),
 
-	TP_printk("stream_tag: %d", __entry->stream_tag)
+	TP_printk("%s: stream_tag: %d", __func__, __entry->stream_tag)
```

Comment:
I'm not sure about this one. Doesn't a tracepoint already record the function name?

Reply:
Thanks @ranj063 , removed.